doc/man3/OSSL_DESERIALIZER_from_bio.pod

```diff
@@ -197,11 +215,11 @@ provides I<deser>.
 =back
 
 At least one of "data" or "reference" must be present, and it's
-possible that both can be.  A I<finalizer> should choose to use the
+possible that both can be.  A constructor should choose to use the
 "reference" parameter if possible, otherwise the "data" parameter.
```

Comment:
otherwise the "data" parameter what?  _is used_, _must be used_ ??

Reply:
The intended meaning is this:

'A constructor should choose to use the "reference" parameter if possible, otherwise [it should use] the "data" parameter.'

Is that not a valid way to construct such a sentence?
(I try not to sprinkle "must" around more than necessary, btw...  after all, we only have so much control over what others are going to do)

Reply:
I inserted "it should use"